jetstream/tests/test_analysis.py

```diff
@@ -558,7 +624,16 @@ def test_create_subset_metric_table_query_univariate_exposures(experiments):
     assert expected_query == actual_query
 
 
-def test_create_subset_metric_table_query_covariate_exposures(experiments, monkeypatch):
```

Comment:
What was the `experiments` parameter doing before? Seems like we've replaced it with a single, custom `Experiment` for each test.

Reply:
The experiments parameter comes from the fixture in conftest. I didn't want to change the existing fixture and potentially mess up other tests using it, but I did need to add the `bucket_config` parameter with a dynamic `randomization_unit` to the experiment used by these tests based on the parametrized randomization_unit parameter.

Another potential approach could be to create a new fixture with an experiment for each of the randomization_units, but then we'd need to loop over them in the tests. I thought this was a bit nicer.

Reply:
Thanks for the explanation!